src/lang/english.txt

```diff
@@ -1618,6 +1618,10 @@ STR_CONFIG_SETTING_RIGHT_MOUSE_BTN_EMU_OFF                      :Off
 
 STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE                        :Close window on right-click: {STRING2}
 STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE_HELPTEXT               :Closes a window by right-clicking inside it. Disables the tooltip on right-click!
+###length 3
+STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE_NO                     :No
+STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE_YES                    :Yes
+STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE_YES_EXCEPT_STICKY      :Yes, except pinned
```

Comment:
In two places it is called `pinned`, while in all other places it is called `sticky`. I think we should go for sticky, as that seems to be the term in use. What do you think?

Reply:
Code seems to call it sticky, while all strings refer to it as pinned. My code was a little inconsistent on that naming scheme so I fix that.

Reply:
I couldn't find any reference to "pinned" in the strings. Did I miss any? (Honest question, to be clear)